mne/decoding/time_gen.py

```diff
@@ -257,6 +257,10 @@ def fit(self, epochs, y=None, picks=None):
         else:
             n_chunk = multiprocessing.cpu_count()
 
+        # Avoid splitting the data in more time chunk than there is time points
+        if n_chunk > X.shape[2]:
+            n_chunk = X.shape[2]
```

Comment:
to add only one line I would I have done

n_chunk = min(n_chunk, X.shape[2])

but it's a nitpick.

+1 for merge when travis is happy